tests/vmi_iothreads_test.go

```diff
@@ -218,6 +218,60 @@ var _ = Describe("IOThreads", func() {
 			// though the limit would have supported 8.
 			table.Entry("[test_id:3096]for four CPUs", 4, 6),
 		)
+
+		// IOThread with Emulator Thread
+
+		It("Should place io and emulator threads on the same pcpu with auto ioThreadsPolicy", func() {
+			policy := v1.IOThreadsPolicyAuto
+			vmi.Spec.Domain.IOThreadsPolicy = &policy
+			vmi.Spec.Domain.CPU = &v1.CPU{
+				Cores:                 1,
+				DedicatedCPUPlacement: true,
+				IsolateEmulatorThread: true,
+			}
+			vmi.Spec.Domain.Resources = v1.ResourceRequirements{
+				Requests: k8sv1.ResourceList{
+					k8sv1.ResourceMemory: resource.MustParse("64M"),
+				},
+			}
+
+			tests.AddEphemeralDisk(vmi, "ded2", "virtio", tests.ContainerDiskFor(tests.ContainerDiskCirros))
```

Comment:
Please rename references to "ded2" to something like "disk1" ... in this test case, the disk isn't flagged as dedicated, so it's confusing me.

Reply:
actually, it might be helpful to actually create a disk requesting dedicated IOthreads--just to prove they're also still isolated.

Reply:
I've added another disk with a dedicated iothread, to show that it will have a separate thread on the isolated pCPU.
Thanks @stu-gott